### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by innat**
# Description

- Add colab and kaggle badges.

## Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [ ] My code follows the [pre-commit style and check guidelines](https://openvinotoolkit.github.io/anomalib/guides/using_pre_commit.html#pre-commit-hooks) of this project.
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing tests pass locally with my changes

### CODE REVIEW

**Review by samet-akcay — APPROVED** [review]
Thanks!

File: README.md [hunk 1]
```diff
@@ -45,8 +45,9 @@ To get an overview of all the devices where `anomalib` as been tested thoroughly
 
 For getting started with a Jupyter Notebook, please refer to the [Notebooks](./notebooks) folder of this repository. Additionally, you can refer to a few created by the community:
 
-- [Google Colab](https://colab.research.google.com/drive/1K4a4z2iZGBNhWdmt9Aqdld7kTAxBfAmi?usp=sharing) by @bth5
-- [Kaggle](https://www.kaggle.com/code/ipythonx/mvtec-ad-anomaly-detection-with-anomalib-library) by @innat
+<a href="https://colab.research.google.com/drive/1K4a4z2iZGBNhWdmt9Aqdld7kTAxBfAmi?usp=sharing" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a> by @bth5
```
Comment (ashwinvaidya17) [inline README.md:48]: While we are at it maybe it would be nice to link both the GitHub profiles as well instead of `@username`

**Comment by ashwinvaidya17** [comment]
Not in the scope of this PR but it would be nice if you (@innat and @bth5) fix the anomalib version/commit in your notebooks to ensure that they work in the future as well 😃

**Review by ashwinvaidya17 — APPROVED** [review]
Minor comments but I am fine if we merge with current change.

File: README.md (hunk 1 above)
Comment (innat) [reply README.md]: DONE.

**Pull request merged**